src/main/java/publishers/JiraPublisher.java

```diff
@@ -29,6 +29,8 @@
 
 	private HttpURLConnection connection;
 	private String urlParameters;
+	private HttpURLConnection connection2;
+	private String urlParameters2;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "urlParameters2" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)